<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainQT</class>
 <widget class="QMainWindow" name="mainQT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ポリゴンの切れ</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/imgs/icon.png</normaloff>:/imgs/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>351</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QListWidget" name="sagasu_list">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>331</width>
          <height>411</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
       </widget>
       <widget class="QLineEdit" name="kotoba_line">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>271</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QPushButton" name="sagasu_btn">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>10</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/imgs/search.png</normaloff>:/imgs/search.png</iconset>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>361</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QTextBrowser" name="naiyou">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>341</width>
          <height>411</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="mp3_btn">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>430</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/imgs/mp3.png</normaloff>:/imgs/mp3.png</iconset>
        </property>
       </widget>
       <widget class="QPushButton" name="add_book_btn">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>430</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/imgs/bookmarkadd.png</normaloff>:/imgs/bookmarkadd.png</iconset>
        </property>
       </widget>
       <widget class="QPushButton" name="add_kotoba_btn">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>430</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/imgs/add.png</normaloff>:/imgs/add.png</iconset>
        </property>
       </widget>
       <widget class="QPushButton" name="del_kotoba_btn">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>430</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/imgs/del.png</normaloff>:/imgs/del.png</iconset>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>0</y>
      <width>351</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QFrame" name="frame_3">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QTabWidget" name="tabWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>361</width>
          <height>491</height>
         </rect>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Triangular</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="widget">
         <attribute name="title">
          <string>ブックマーク</string>
         </attribute>
         <widget class="QLCDNumber" name="bookmark_count_lcd">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>10</y>
            <width>211</width>
            <height>41</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="del_book_btn">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>10</y>
            <width>41</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/imgs/del.png</normaloff>:/imgs/del.png</iconset>
          </property>
         </widget>
         <widget class="QListWidget" name="bookmark_list">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>331</width>
            <height>391</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="widget">
         <attribute name="title">
          <string>検索記録</string>
         </attribute>
         <widget class="QPushButton" name="history_reset_btn">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>420</y>
            <width>41</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/imgs/reset.png</normaloff>:/imgs/reset.png</iconset>
          </property>
         </widget>
         <widget class="QTextBrowser" name="history_view">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>331</width>
            <height>401</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="widget">
         <attribute name="title">
          <string>統計</string>
         </attribute>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>20</y>
            <width>171</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>登録された見出し語の数</string>
          </property>
         </widget>
         <widget class="QLCDNumber" name="count_lcd">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>50</y>
            <width>241</width>
            <height>41</height>
           </rect>
          </property>
         </widget>
         <widget class="QWidget" name="verticalLayoutWidget_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>130</y>
            <width>331</width>
            <height>301</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="layer"/>
         </widget>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
